### ISSUE TITLE
Refresh SecureDrop team Qubes dev environment

### ISSUE THREAD

**Issue by eloquence** [freedomofpress/securedrop-workstation#74]
This is an FPF task to track whether all members of the SecureDrop team have an up-to-date development environment (Qubes 4.0 final or later, working SecureDrop workstation that can replicate the full document download flow).

Please only check the box for your name once you can confirm both of the above.

4/4-4/18 + 4/18-5/2 sprint commitments:
- [x] Jen
- [x] Kushal
- [x] Mickael
- [x] Conor 
- [x] Emmanuel
- [X] Erik
- [x] Mike

TBD:  Freddy (optional), Harris (pending SD website launch)

**Comment by conorsch** [comment]
> Qubes 4.0 final or later

Still on rc4 over here. The repo README still recommends installing an rc version of Qubes, which we should update. 

Additionally, existing rc installs will need to _disable_ the testing repos in order to upgrade fully to 4 stable. We should document those steps here, so that folks know exactly which files to change before checking the box in the OP here. The [Qubes 4 release announcement](https://www.qubes-os.org/news/2018/03/28/qubes-40/) mentions this briefly:

> current users of 4.0-rc5 can upgrade in-place by downloading the latest updates from the stable repositories in both dom0 and TemplateVMs.

but more explicit steps for devs here would be grand. Bonus: adding tests to validate the stable/non-rc settings in dom0 and VMs would be fantastic, and enable folks to run the test suite to confirm compliance with the 4 stable requirement.

**Comment by conorsch** [comment]
Checking my box. Specifically, the actions I performed were:

* Upgrade BIOS on Qubes 4 workstation
* Install Qubes 4 fresh from ISO
* Pulled down workstation repo
* Configured `config.json`
* Ran `make all` ; no errors
* Ran `make test`; no errors, but got some stderr output that should be followed up on (will open separate ticket)

Also working on providing more info for the discussion in #79.

**Comment by msheiny** [comment]
I'm up and running now. One note is that I did hit #86 which was confusing at first. Will follow up in that ticket for more deets.

**Comment by edenemmanuel** [comment]
I can confirm usage of the SecureDrop Workstation (powered by Qubes 4.0) works.

Specifically I did the following:
* Installed a fresh Qubes 4.0 (after verifying its authenticity via OpenPGP etc.)
* Updated dom0 and the Fedora 26 Templates to the latest updates (and installed `ntp` in Fedora 26)
* Set up my SecureDrop Workstation dev AppVM
* Configured environment with my instance specific settings (via `config.json` etc.)
* Ran `make all`. No errors. :tada: 
* Ran `make test`. No errors, but saw similar output to Conor.
* Ran through a journalist downloading docs from multiple sources flow. Confirmed successful opening across various VMs.

Do let me know if you'd like me to test more stuff. Happy to do so.
